backend/src/main/java/com/capstone/moneytree/controller/StockController.java

```diff
@@ -94,4 +92,13 @@ public StockController(StockMarketDataService stockMarketDataService) {
       Logo logo = stockMarketDataService.getLogo(symbol);
       return ResponseEntity.ok(logo);
    }
+
+   /*
+   Valid ranges: [1d, 5d, 1mo, 3mo, 6mo, 1y, 2y, 5y, 10y, ytd, max]
+   Valid intervals: [1m, 2m, 5m, 15m, 30m, 60m, 90m, 1h, 1d, 5d, 1wk, 1mo, 3mo]
+   */
+   @GetMapping(value = "/yahoochart/{symbol}", produces = {"application/json"})
```

Comment:
Do we absolutely need ```produces = {"application/json"}``` here? I thought the output was JSON by default.

Reply:
In this case yes, since it's returning a String (formatted as json), the content-type of the header is text/html so the explicit json header is required. 
![Screenshot from 2021-02-19 19-18-12](https://user-images.githubusercontent.com/33789241/108575230-89cf2d00-72e7-11eb-9c57-41eb6302c12b.png)

Reply:
Ahh I see. Thanks!